src/components/DatePicker/DatePicker.module.scss

```diff
@@ -0,0 +1,632 @@
+    }
+  }
+  
+  .react-datepicker__month-container {
+    float: left;
+  }
+  
+  .react-datepicker__year {
+    margin: 0.4rem;
+    text-align: center;
+    &-wrapper {
+      display: flex;
+      flex-wrap: wrap;
+      max-width: 180px;
+    }
+    .react-datepicker__year-text {
+      display: inline-block;
+      width: 4rem;
+      margin: 2px;
+    }
+  }
+  
+  .react-datepicker__month {
+    margin: 0.4rem;
+    text-align: center;
+
+    .react-datepicker__month-text,
+    .react-datepicker__quarter-text {
+      display: inline-block;
+      padding: var(--size-spacing-xs);
+      margin: 2px;
+    }
+  }
+  
+  .react-datepicker__time-container {
+    float: right;
+    border-left: 1px solid var(--color-brand-grey-lighter);
+    width: 85px;
+  
+    &--with-today-button {
+      display: inline;
+      border: 1px solid #aeaeae;
+      border-radius: 0.3rem;
+      position: absolute;
+      right: -72px;
+      top: 0;
+    }
+  
+    .react-datepicker__time {
+      position: relative;
+      border-bottom-right-radius: 0.3rem;
+      margin-top: var(--size-spacing-xs);
+  
+      .react-datepicker__time-box {
+        width: 85px;
+        overflow-x: hidden;
+        margin: 0 auto;
+        text-align: center;
+        border-bottom-right-radius: 0.3rem;
+        ul.react-datepicker__time-list {
+          list-style: none;
+          margin: 0;
+          height: calc(195px + (1.7rem / 2));
+          overflow-y: scroll;
+          padding-right: 0px;
+          padding-left: 0px;
+          width: 100%;
+          box-sizing: content-box;
+  
+          li.react-datepicker__time-list-item {
+            height: 30px;
+            padding: 5px 10px;
+            white-space: nowrap;
+            transition-duration: 0.2s;
+            transition-property: background-color;
+
+            &:hover {
+              cursor: pointer;
+              background-color: var(--color-brand-grey-lightest);
+            }
+
+            &--selected {
+              background-color: var(--color-brand-primary-base);
+              color: var(--color-brand-white-base);
+              font-weight: bold;
+
+              &:hover {
+                background-color: var(--color-brand-primary-600);
+              }
+            }
+
+            &--disabled {
+              color: var(--color-brand-grey-lightest);
+  
+              &:hover {
+                cursor: default;
+                background-color: transparent;
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+  
+  .react-datepicker__week-number {
+    color: var(--color-brand-grey-lighter);
+    display: inline-block;
+    width: 1.7rem;
+    line-height: 1.7rem;
+    text-align: center;
+    margin: 0.166rem;
+
+    &.react-datepicker__week-number--clickable {
+      cursor: pointer;
+
+      &:hover {
+/*         border-radius: var(--size-border-radius-lg); */
+        background-color: var(--color-brand-grey-lighter);
+      }
+    }
+  }
+  
```

Comment:
should add styles for `react-datepicker__day--outside-month` so that they are dimmer than current month

Reply:
Hey @nathanyoung can you clarify this one? Are we talking about dimmer background color when days outside of the current month are selected?